src/main/java/com/rocketden/main/controller/v1/UserSocketController.java

```diff
@@ -1,30 +1,34 @@
 package com.rocketden.main.controller.v1;
 
-import com.rocketden.main.dto.user.UserDto;
 import java.util.HashSet;
 import java.util.Set;
 
+import com.rocketden.main.dto.user.UserDto;
+
+import org.springframework.messaging.handler.annotation.DestinationVariable;
 import org.springframework.messaging.handler.annotation.MessageMapping;
 import org.springframework.messaging.handler.annotation.SendTo;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class UserSocketController extends BaseRestController {
 
-    Set<UserDto> users = new HashSet<>();
+    private static final String SEND_TO_PATH = BaseRestController.BASE_SOCKET_URL + "/{roomId}/subscribe-user";
+
+    private Set<UserDto> users = new HashSet<>();
 
-    @MessageMapping("/add-user")
-    @SendTo(BaseRestController.BASE_SOCKET_URL + "/subscribe-user")
-    public Set<UserDto> addUser(String nickname) {
+    @MessageMapping("/{roomId}/add-user")
+    @SendTo(SEND_TO_PATH)
+    public Set<UserDto> addUser(@DestinationVariable String roomId, String nickname) {
```

Comment:
This (and `deleteUser(...)`) won't be used anymore, right? If so, should we remove them in a separate PR, or this one?

Reply:
I wasn't sure if we still needed them or not, but yeah I can delete them (I'll probably do it in #41 as this one still technically supports the addUser/deleteUser functionality whereas that one makes breaking changes).